### PR TITLE
Mekhi/about: Created about page

### PR DESCRIPTION
**Pull request by MekhiHart**

### CODE REVIEW

File: client/src/app/about/page.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          backgroundColor: "white",
+        }}
+      >
+        <Typography
+          className="bottom-pad"
+          variant="h3"
+          sx={{
+            fontSize: 45,
+          }}
+        >
```
Comment (Serranoda) [inline client/src/app/about/page.tsx:28]: Change text to align to center. would give page a more centered look

File: client/src/app/about/page.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          often feels like navigating a black box. As a couple of college
+          students, we knew it all too well.
+        </Typography>
+        <Typography className="bottom-pad" variant="h5">
+          That’s why, in 2024, we began SqueakPeek to allow you to connect with
+          other job applicants, share experiences, and track your progress every
+          step of the way. Through message threads, you can communicate about
+          opportunities, discuss interview insights, and feel more informed as
+          you move forward in your career.
+        </Typography>
```
Comment (Serranoda) [inline client/src/app/about/page.tsx:44]: Our about us page content should be structured like the following

Short intro to our team

Our mission goals 

(optional) = Call to action and or Team Image

The text needs some modification, it is too informal

File: client/src/app/about/page.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+          color: "#09556C",
+        }}
+      >
+        <Typography
+          sx={{
+            fontSize: 24,
+            color: "#09556C",
+          }}
+        >
+          Get Started Today
```
Comment (Serranoda) [inline client/src/app/about/page.tsx:67]: About us page is only for gathering information on the team behind the product. No need to add a get started link here.

**Review by Serranoda — CHANGES_REQUESTED** [review]
Page is almost complete. Ive suggested some simple format and content changes

File: client/src/app/about/page.tsx (hunk 2 above)
Comment (MekhiHart) [reply client/src/app/about/page.tsx:44]: This is what we have designed from the figma file? I don't see what's the issue on my end

File: client/src/app/about/page.tsx (hunk 3 above)
Comment (MekhiHart) [reply client/src/app/about/page.tsx:67]: this is what we have designed from the figma file.

File: client/src/app/about/page.tsx (hunk 1 above)
Comment (MekhiHart) [reply client/src/app/about/page.tsx:28]: This was not how it was designed on the figma file.
image

File: client/src/app/about/page.tsx (hunk 2 above)
Comment (Serranoda) [reply client/src/app/about/page.tsx:44]: You are correct. The figma needs to be updated then

File: client/src/app/about/page.tsx (hunk 1 above)
Comment (haophn01) [reply client/src/app/about/page.tsx:28]: This looks good

**Review by haophn01 — APPROVED** [review]

File: client/src/app/about/page.tsx (hunk 3 above)
Comment (haophn01) [reply client/src/app/about/page.tsx:67]: Okie

**Review by haophn01 — APPROVED** [review]
All look good

**Pull request closed without merging**